### ISSUE TITLE
Missing `AuthenticationFinished` resultCode from PaymentResult.ResultCodeEnum

### ISSUE THREAD

**Issue by pharalia** [Adyen/adyen-java-api-library#197]
**Java version**: 1.8.x
**Library version**: 2.1.0
**Description**
The resultCode `AuthenticationFinished` is not present in `PaymentResult.ResultCodeEnum` for mapping when a completed 3DS challenge is submitted to `/authorise3ds2` with the `authenticationOnly` flag set.

```javascript
// Request
{
    "threeDS2Result": {
        "transStatus": "Y"
    },
    "threeDS2Token": "-",
    "merchantAccount": "-",
    "applicationInfo": {
        "adyenLibrary": {
            "name": "adyen-java-api-library",
            "version": "2.1.0"
        }
    }
}
```
```javascript
// Response
{
    "additionalData": {
        "threeds2.threeDS2Result.dsTransID": "-",
        "threeds2.threeDS2Result.eci": "05",
        "threeds2.threeDS2Result.threeDSServerTransID": "-",
        "threeds2.threeDS2Token": "-",
        "threeds2.threeDS2Result.authenticationValue": "-",
        "threeds2.threeDS2Result.transStatus": "Y"
    },
    "pspReference": "-",
    "resultCode": "AuthenticationFinished"
}
```

**Comment by martinsrenato** [comment]
Hi @pharalia!

Thank you for the PR. We are going to review and merge it ASAP.

Cheers,
Renato
Adyen

**Comment by martinsrenato** [comment]
Hi @pharalia!

We approved your PR and released it on version 2.3.1.
Thanks for your contribution :)

Cheers,
Renato
Adyen
